plutus-tx-plugin/test/Blueprint/Tests.hs

```diff
@@ -39,35 +51,42 @@ contractBlueprint =
             preambleLicense = Just "MIT"
           },
       contractValidators =
-        Set.singleton
-          MkValidatorBlueprint
-            { validatorTitle = "Acme Validator",
-              validatorDescription = Just "A validator that does something awesome",
-              validatorParameters =
-                Just $
-                  pure
-                    MkParameterBlueprint
-                      { parameterTitle = Just "Acme Parameter",
-                        parameterDescription = Just "A parameter that does something awesome",
-                        parameterPurpose = Set.singleton Purpose.Spend,
-                        parameterSchema = definitionRef @Fixture.Params
-                      },
-              validatorRedeemer =
-                MkArgumentBlueprint
-                  { argumentTitle = Just "Acme Redeemer",
-                    argumentDescription = Just "A redeemer that does something awesome",
-                    argumentPurpose = Set.fromList [Purpose.Spend, Purpose.Mint],
-                    argumentSchema = definitionRef @Fixture.Redeemer
-                  },
-              validatorDatum =
-                Just
-                  MkArgumentBlueprint
-                    { argumentTitle = Just "Acme Datum",
-                      argumentDescription = Just "A datum that contains something awesome",
-                      argumentPurpose = Set.singleton Purpose.Spend,
-                      argumentSchema = definitionRef @Fixture.Datum
-                    },
-              validatorCompiledCode = Just Fixture.serialisedScript
-            },
+        Set.fromList
+          [ MkValidatorBlueprint
+              { validatorTitle =
+                  "Acme Validator #1",
+                validatorDescription =
+                  Just "A validator that does something awesome",
+                validatorParameters =
+                  Just . pure $
+                    $(deriveParameterBlueprint ''Fixture.Params (Set.singleton Purpose.Spend)),
```

Comment:
I'm unsure if this is good. People generally dislike TH and IME they're even less comfortable with it in expression positions. I guess it's cheap enough to have as an option.